src/OrleansProviders/Streams/AzureQueue/AzureQueueAdapterFactory.cs

```diff
@@ -32,8 +32,8 @@ namespace Orleans.Providers.Streams.AzureQueue
 {
     public class AzureQueueAdapterFactory : IQueueAdapterFactory
     {
-        private const string CACHE_SIZE = "CacheSizeKB";
-        private const int DEFAULT_CACHE_SIZE = 4 * (1 << 10);
+        private const string CACHE_SIZE = "CacheSize";
```

Comment:
It used to be KB, which was confusing and out of place. It should be just a size -  just like when you pass capacity to the List: a max number of items in the list. Same here: capacity of the cache - max number of items in the cache.
The KB was totally unrelated, and this PR removing this confusion.